core/src/main/java/org/apache/calcite/sql/validate/SqlConformance.java

```diff
@@ -408,4 +408,14 @@
    * false otherwise.
    */
   boolean allowPluralTimeUnits();
+
+  /**
+   * Whether to allow qualifying common column.
+   *
+   * <p>Among the built-in conformance levels, false in
```

Comment:
What do you mean by "common column" ? Can you clarify ?

Reply:
Will do.